File: Assignment12/Question3.c
#include<stdio.h>
void pattern(int iNo)
{
    int iCnt=0;
    for(iCnt=iNo;iCnt>0;iCnt--)
    {
        printf("%d   *  ",iCnt);
    }
    printf("\n");
}
int main()
{
    int iValue=0;
    printf("Enter number:");
    scanf("%d",&iValue);

    pattern(iValue);
    return 0;
}

File: Assignment7/Question3.java
import java.util.*;
import java.lang.*;
class Check
{
    int CountOdd(int iNo)
    {
        int iCnt=0;
        while (iNo>0) {
    
            int iDigit = iNo%10;
            if((iDigit>=3) && (iDigit<=7))
            {
                iCnt++;
            }
            iNo = iNo/10;
        }       
    return iCnt;
    }

}

class Question3{
    public static void main(String args[]) {
    
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number:");
        int iValue = sobj.nextInt();
        Check cobj = new Check();
        int iRet = cobj.CountOdd(iValue);
        System.out.printf("%d Number between 3-7.\n",iRet);
    }
}